src/main/java/com/jackbaretto/scrumtest/extractor/ExtractionParameters.java

```diff
@@ -6,12 +6,12 @@
 import java.util.Set;
 
 /**
- * Set up parameters for MCQ file(s) extraction.
+ * Set up parameters for MCQ file(s) extraction..
  * Created by florentsailly on 14/12/2016.
  */
 public class ExtractionParameters {
 
-    private final Set<File> mcqPictures = new HashSet<File>();
+    private final Set<File> mcqPictures = new HashSet<>();
```

Comment:
Since Java 7, don't use diamond operator is considered clumsy (https://dev.eclipse.org/sonar/rules/show/squid:S2293).